src/main/webapp/open_hours.js

```diff
@@ -0,0 +1,82 @@
+document.addEventListener("DOMContentLoaded", async function() {
```

Comment:
Add a top level comment to what this class should do, what it expects to be present in the DOM, etc.